docs/docs/README.md

```diff
@@ -33,9 +53,9 @@ The diagram below provides a high level overview of what a typical deployment wi
 
 Atlassian Data Center products that can be provisioned with this tool:
 
-|  Bamboo DC                                                                                         |
-|----------------------------------------------------------------------------------------------------|
-| [8.1](https://confluence.atlassian.com/bamboo/bamboo-8-1-release-notes-1103070461.html){.external}  |
+|  Bamboo DC                                                                                         | Confluence DC                                                                                          | Jira DC                                                                                                                  |
+|----------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| [8.1](https://confluence.atlassian.com/bamboo/bamboo-8-1-release-notes-1103070461.html){.external} | [7.13](https://confluence.atlassian.com/doc/confluence-7-13-release-notes-1044114085.html){.external}  | [8.19](https://confluence.atlassian.com/jirasoftware/jira-software-8-19-x-release-notes-1082526044.html){.external}      |  
```

Comment:
Do we want to use the LTS for Jira? i.e. 8.20.x.

Reply:
We're reflecting what is [documented in the data-center-helm-charts ](https://atlassian.github.io/data-center-helm-charts/#product-versions) i.e. "The minimum versions that we support for each product are:" This makes me wonder, should the heading for this section be updated from:

`Atlassian Data Center products that can be provisioned with this tool:`

to

`The minimum versions that we support for each product are:`